happi/item.py

```diff
@@ -348,6 +349,15 @@ def show_info(self, handle=sys.stdout):
         for key in public_keys:
             pt.add_row([key, show_info[key]])
 
+        # initialize table so we can grab widths
+        _ = pt.get_string()
+
+        # set width of Value column dynamically
+        # set width(Value) = terminal width - width(EntryInfo column)
+        entry_info_width = pt._widths[0] + 10  # account for padding
+        term_width = os.get_terminal_size()[0]
+        pt._max_width = {'Value': max(60, term_width - entry_info_width)}
```

Comment:
80 is the "standard" terminal width, if there is one. Why 60?

Reply:
I'm only modifying the width of the `Value` column, so it was originally 80 - (estimated width of the EntryInfo column)